src/list/list.model.ts

```diff
@@ -0,0 +1,19 @@
+import { prop } from '@typegoose/typegoose';
+import { TimeStamps } from '@typegoose/typegoose/lib/defaultClasses';
+import { ObjectId } from 'mongodb';
+import * as MUUID from 'uuid-mongodb';
+import { IListModel } from './list.interfaces';
```

Comment:
Please, add blank line after node_modules definitions and before files.

Reply:
the blank line was added